pkgs/development/libraries/libosinfo/default.nix

```diff
@@ -20,6 +20,8 @@ stdenv.mkDerivation rec {
 
   patches = [
     ./osinfo-db-data-dir.patch
+    ./0001-tests-Expand-the-arch-s-parser-for-isodetect.patch
+    ./0002-db-Force-anchored-patterns-when-matching-regex.patch
```

Comment:
Let's fetchpatch these from upstream gitlab rather than putting them in our repo.
https://gitlab.com/libosinfo/libosinfo/commit/e6168463f4fc659b9827b5c8694dc1c6d7d5239a.patch
https://gitlab.com/libosinfo/libosinfo/commit/b9cb227842948b1b2289cdd3e9b8d925664c2ee7.patch